boilerplate/android/app/src/main/java/com/helloworld/MainActivity.java

```diff
@@ -28,6 +30,12 @@ protected ReactActivityDelegate createReactActivityDelegate() {
     return new ReactActivityDelegateWrapper(this, new MainActivityDelegate(this, getMainComponentName()));
   }
 
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    RNBootSplash.init(this);            // <- initialize the splash screen
+    super.onCreate(savedInstanceState); // or super.onCreate(null) with react-native-screens
```

Comment:
@frankcalise do we wanna do null here?

Reply:
@yulolimum yeah my bad I had did it locally in a test project - thanks!